blivet/formats/fs.py

```diff
@@ -627,6 +628,9 @@ def _teardown(self, **kwargs):
 
         udev.settle()
 
+    def change_mountpoint(self, dry_run=False):
```

Comment:
The support for changing mountpoint with `ActionConfigureFormat` can be a separate commit, but it should be finished before the PR is set to be ready to be merged.

Reply:
Added an explanation comment to make the intention clearer.

Reply:
The point of this is to be able to revert any mountpoint changes in case the action gets cancelled. Since it is just a property value change, it does not need any code.
I am open to suggestions but this came to me as a simple solution that utilizes structures already in place.